@xen-orchestra/proxy-cli/package.json

```diff
@@ -32,6 +32,7 @@
     "@iarna/toml": "^2.2.0",
     "@vates/read-chunk": "^0.1.1",
     "app-conf": "^0.7.0",
+    "content-type": "^1.0.4",
```

Comment:
Why is it necessary?

Reply:
Please see https://github.com/vatesfr/xen-orchestra/blob/b89cddf194b208dac02580067e9b8b4486692eee/%40xen-orchestra/proxy-cli/src/index.js#L101

Reply:
This does not answer my question, why can't you simply do `headers['content-type'] === 'application/octet-stream'`.

Reply:
`content-type` as definition can contains more than the response MIME type.

Reply:
Fine by me.